packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/DrawingToolMarks/DrawingToolMarksConnector.js

```diff
@@ -1,14 +1,14 @@
 import { observer } from 'mobx-react'
 import { forwardRef } from 'react'
-import { useFreehandLineReductions } from '@hooks'
+import { useMachineLearntReductions } from '@hooks'
 import DrawingToolMarks from './DrawingToolMarks'
 
-function withFreehandLineReductions(Component) {
-  function WithFreehandLineReductions(props, ref) {
-    const freehandLineProps = useFreehandLineReductions()
+function withMachineLearntReductions(Component) {
+  function WithMachineLearntReductions(props, ref) {
+    const freehandLineProps = useMachineLearntReductions()
```

Comment:
Similar note on naming. With these changes, `freehandLineProps` is no longer just freehand line right? It could be any drawing tool.